RELEASE-NOTES.md

```diff
@@ -32,7 +32,7 @@
 
   - [Added Stein-Variational Gradient Descent as well as Amortized SVGD (experimental).](https://github.com/pymc-devs/pymc3/pull/2183)
 
-  - [Added pm.generator() to easily specify mini-batches.](http://pymc-devs.github.io/pymc3/notebooks/bayesian_neural_network_opvi-advi.html#Minibatch-ADVI)
+  - [Added `Minibatch` to easily specify mini-batches.](http://pymc-devs.github.io/pymc3/notebooks/bayesian_neural_network_opvi-advi.html#Minibatch-ADVI)
```

Comment:
It is not used in this notebook... I think reference to VI intro is more suitable